rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.74.1"
+channel = "stable"
```

Comment:
@Alw3ys although an unrelated change, believe this could be helpful to always stay on the latest stable release.

Reply:
I strongly believe is best to pin versions everywhere and setup a task to do it once a while. otherwise people will have different versions